Projects/2-Intermediate/celestia.md

```
@@ -0,0 +1,40 @@
+
+## Developer Notes
+-By using the credentials created by super admin then only the doctor can access the records of his patients
+-The doctor login consists of two-step verification by completing it. Once completed, they can login through the application
+
+## Bonus features
+**Treatment Recommendations**: AI-generated treatment summaries including self-care tips
+**AI-Powered Analysis**: Get AI-generated condition analysis based on symptoms
+**Medical Conditions Database**: Comprehensive information on various medical conditions
+
+## Useful links and resources
+**Supabase** (for database): https://supabase.com/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)